apps/fxc-front/src/app/logic/track.ts

```diff
@@ -1,9 +1,21 @@
-import { extractGroupId } from '@flyxc/common';
+import {extractGroupId, RuntimeTrack} from '@flyxc/common';
 
 import { unwrapResult } from '@reduxjs/toolkit';
 
-import { store } from '../redux/store';
-import { fetchTrack } from '../redux/track-slice';
+import {AppDispatch, store} from '../redux/store';
```

Comment:
The formatting does not look consistent here.

Nx has an action to format the code.

I periodically use 
```
nx run-many --target=lint,build,test,format
```
when I develop. Nx has a nice cache that would only re-run what's needed.
It would catch a few missing `;` here and there at least.

Reply:
formating and linting should be ok now (I hope so)